Yelp/Business.swift

```diff
@@ -91,4 +91,9 @@ class Business: NSObject {
     class func searchWithTerm(term: String, sort: YelpSortMode?, categories: [String]?, deals: Bool?, completion: ([Business]!, NSError!) -> Void) -> Void {
         YelpClient.sharedInstance.searchWithTerm(term, sort: sort, categories: categories, deals: deals, completion: completion)
     }
+    class func searchWithTermAndLocation(term: String, location: String, completion: ([Business]!, NSError!) -> Void) {
+        
+        YelpClient.sharedInstance.searchWithTermAndLocation(term, location: location, completion: completion)
+    }
+    
 }
```

Comment:
Can you fix the whitespace here? There should be a space between functions, and there shouldn't be a space in the first line of the function.